src/output-json-alert.c

```diff
@@ -810,6 +812,7 @@ static void SetFlag(const ConfNode *conf, const char *name, uint16_t flag, uint1
 static void JsonAlertLogSetupMetadata(AlertJsonOutputCtx *json_output_ctx,
         ConfNode *conf)
 {
+    static bool _warn_no_meta = false;
```

Comment:
same here, why the leading _?

Reply:
Following convention: https://github.com/OISF/suricata/blob/master/src/output-json-anomaly.c#L375:L376

Would you like the `_` removed from both files?